nemo/collections/nlp/modules/common/transformer/transformer_generators.py

```diff
@@ -301,7 +301,8 @@ def _forward(
         scores, prefixes = scores.view(-1, 1), prefixes.view(-1, 1)
 
         # repeat init target prefixes and cached memory states beam_size times
-        prefixes = torch.cat((tgt.repeat(1, self.beam_size).view(-1, 1), prefixes), dim=1)
+        # prefixes = torch.cat((tgt.repeat(1, self.beam_size).view(-1, 1), prefixes), dim=1)
```

Comment:
remove

Reply:
@stevehuang52 move to the copy in asr collection

Reply:
done